### PR TITLE
add minutes from the overflow slot

### PR DESCRIPTION
**Pull request by matfechner**

### CODE REVIEW

**Comment by berendt** [comment]
@matfechner Can you please add me to the list of attendees

File: iaas/20230220.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Overflow session
+## Participants (preferably github handles)
+* @fkr
+* @tibeer
+* @ra-beer
+* @ppkuschy
+* @matfechner
+
```
Comment (ppkuschy) [inline iaas/20230220.md:8]:
```suggestion
* @berendt 
```
@berendt Wants to be added as attendee

**Review by ppkuschy — CHANGES_REQUESTED** [review]
Pls see suggestion

**Review by fkr — APPROVED** [review]
LGTM!

**Review by ppkuschy — APPROVED** [review]

**Pull request merged**